service/src/test/java/org/mskcc/cmo/metadb/service/SampleServiceTest.java

```diff
@@ -102,17 +104,28 @@ public void initializeMockDatabase() throws Exception {
                 .get("mockIncomingRequest5JsonPtMultiSamples");
         MetadbRequest request5 = RequestDataFactory.buildNewLimsRequestFromJson(request5Data.getJsonString());
         requestService.saveRequest(request5);
+        
+        final ObjectMapper mapper = new ObjectMapper();
 
-        //persist mock clinical data
-        MockJsonTestData clinicalSample = mockDataUtils.mockedDmpMetadataMap
+        //mock clinical data dmp id: P-0000001-N01-IM3
+        MockJsonTestData clinicalSample1 = mockDataUtils.mockedDmpMetadataMap
                 .get("P0000001N01IM3");
-        final ObjectMapper mapper = new ObjectMapper();
-        DmpSampleMetadata dmpSample = mapper.readValue(clinicalSample.getJsonString(),
+        DmpSampleMetadata dmpSample1 = mapper.readValue(clinicalSample1.getJsonString(),
                 DmpSampleMetadata.class);
-        MetadbSample clinicalSample1 = SampleDataFactory.buildNewClinicalSampleFromMetadata(
-                dmpSample.getDmpPatientId(), dmpSample);
-        sampleService.saveMetadbSample(clinicalSample1);
-
+        String cmoPatientId = mockDataUtils.mockedDmpPatientMapping.get(dmpSample1.getDmpPatientId());
+        MetadbSample builtClinicalSample1 = SampleDataFactory.buildNewClinicalSampleFromMetadata(
+                cmoPatientId, dmpSample1);
+        sampleService.saveMetadbSample(builtClinicalSample1);
+        
+      //mock clinical data dmp id: P-7778999-N01-IM3
+        MockJsonTestData clinicalSample2 = mockDataUtils.mockedDmpMetadataMap
```

Comment:
something else I realized is that these clinical samples aren't being persisted correctly so I'll make a PR to fix that and to also load and persist _all_ of the mocked clinical samples